qemu/tests/cfg/multi_host.cfg

```diff
@@ -33,32 +37,12 @@
                 - fd:
                 - exec:
 
-            variants booting_phase:
-                #Time when start migration
-                - after_login_vm:
-                    paused_after_start_vm = no
-                    need_to_login = yes
-                    preprocess_env = no
-                - early_boot_vm:
-                    no measure_migration_speed
-                    login_timeout = 420
-                    paused_after_start_vm = yes
-                    preprocess_env = yes
-                    variants:
-                        - timeout_0:
-                            start_migration_timeout = 0
-                        - timeout_6:
-                            start_migration_timeout = 6
-                        - timeout_random:
-                            start_migration_timeout = random
-                            min_random_timeout = 5
-                            max_random_timeout = 20
-
             variants migraiton_type:
                 - mig_online:
                     mig_offline = no
                 - mig_offline:
                     mig_offline = yes
+                    only host_mig_offline
```

Comment:
1. Suggest to remove the variant, and set offline cfg in host_mig_offline case.
otherwise we need to set only mig_offline and host_mig_offline each other
             variants migraiton_type:
                 - mig_online:
                     mig_offline = no
                 - mig_offline:
                     mig_offline = yes

2. or another method, remove host_mig_offline, and run one basic case on mig_offline